spec/spec_helper.rb

```diff
@@ -149,3 +149,10 @@ def env_args_to_hash(*args)
     Jekyll::GitHubMetadata.logger = Logger.new(StringIO.new) unless ENV["DEBUG"]
   end
 end
+
+def versions
+  require "github-pages"
+  GitHubPages.versions
+rescue LoadError
+  {}
+end
```

Comment:
Ah ok. We have removed the test with `github-pages` so maybe remove this for now with a comment?